modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -0,0 +1,194 @@
+
+|`deviceClasses.name`
+|`string`
+|Specify a name for the LVM volume group (VG). 
+
+If this field is set to the name of a VG from the previous {lvms} installation, {lvms} recreates the resources related to that volume group in the current {lvms} installation. 
+
+You can only recover the VGs but not the logical volume associated with the VG.
+
+[IMPORTANT]
+====
+Before recovering the VGs from the previous {lvms} installation, ensure that the following conditions are met:
+
+* VGs must not be corrupted.
+* VGs must have the `lvms` tag. For more information on adding tags to LVMS objects, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/grouping-lvm-objects-with-tags_configuring-and-managing-logical-volumes#doc-wrapper[Grouping LVM objects with tags].
+====
+
+|`deviceClasses.fstype`
+|`string`
+|Set this field to `ext4` or `xfs`. By default, this field is set to `xfs`.
+
+|`deviceClasses.default`
+|`boolean`
+|Set this field to `true` to indicate that a device class is the default. Otherwise, you can set it to `false`. You can only configure a single default device class. 
+
+
+|`deviceClasses.nodeSelector`
+|`object`
+|Contains the configuration to choose the nodes on which you want to create the LVM volume group. If this field is empty, all nodes without no-schedule taints are considered.
+
+[NOTE]
+====
+On the control-plane node, {lvms} detects and uses the additional worker nodes when the new nodes become active in the cluster. 
+====
+
+Configure the `nodeSelector.nodeSelectorTerms` field with a list of requirements that are used to select the node.
+
+
+|`deviceClasses.deviceSelector`
+|`object`
+|Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group, and force wipe the devices that are added to the LVM volume group. 
+
+You can specify the device paths in the `paths` field, the `optionalPaths` field, or both. If you do not specify the device paths in both `paths` and `optionalPaths`, {lvms} adds the supported unused devices to the LVM volume group.
+
+[IMPORTANT]
+====
+After a device is added to the LVM volume group, it cannot be removed.
+==== 
+
+{lvms} adds the devices to the LVM volume group only if the following conditions are met:
+
+* The device path exists.
+
+* The device is supported by {lvms}.
+
+[NOTE]
+====
+The devices that you want to add to the volume group must be supported by {lvms}. For information about unsupported devices, see _Devices not supported by {lvms}_ in _Additional resources_.
+====
+
+You can also add the path to the RAID arrays to integrate the RAID arrays with {lvms}. For more information, see _Integrating RAID arrays with {lvms}_ in _Additional resources_. 
+
+|`deviceClasses.thinPoolConfig`
+|`object`
+|Contains the configuration to create a thin pool in the LVM volume group.
+
+|====
+
+[discrete]
+=== deviceSelector object configuration
```

Comment:
You might need to check with an SME, but I am pretty certain that it isn't accurate to call a subsection of a CR an "object". We usually talk about the whole CR/resource as an object.

So you might need to rethink how you word this tables for the subsections. You can probably look around the repo for how others do this

But otherwise I think that the tables are much easier to read/scan than the previous format you used

Reply:
Added all the fields in the same table.